tests/open_alchemy/helpers/test_peek/test_peek_extension.py

```diff
@@ -646,35 +59,179 @@ def test_mixins(schema, schemas, expected_mixins):
     assert mixins == expected_mixins
 
 
+@pytest.mark.parametrize(
+    "prefix",
+    [pytest.param(p, id=p) for p in helpers.peek.VALID_PREFIXES],
+)
+@pytest.mark.parametrize(
+    "key_values, func, expected_value",
+    [
+        pytest.param(
+            [(extension, value)],
+            getattr(helpers.peek, extension.replace("-", "_")),
+            expected,
+            id=f"valid {extension}",
+        )
+        for extension, value, expected in [
+            ("index", True, True),
+            ("index", False, False),
+            ("unique", True, True),
+            ("unique", False, False),
+            ("autoincrement", True, True),
+            ("autoincrement", False, False),
+            ("primary-key", True, True),
+            ("primary-key", False, False),
+            ("tablename", "table 1", "table 1"),
```

Comment:
I know this wasn't there before, given how easy this is now, it might be worth adding one alternative case so that we can be sure that the function isn't just returning a static "table 1", e.g., someone made it static just to get something to work but then forgot to undo it.

Reply:
Addressed ✅